pkg/iris/scan.go

```diff
@@ -88,7 +91,22 @@ func (s *scanner) Scan(getKubernetesAPI kubernetes.GetKubernetesAPI, config *res
 		logger.Errorf(StatusErrorFormat, s.runId, err)
 		return err
 	}
-	mapper := NewMapper(kubernetesAPI, kubernetesConfig.Cluster, s.workspaceId, kubernetesConfig.BlackListedNamespaces, s.runId)
+
+	if viper.GetBool(utils.WorkloadFlag) {
```

Comment:
If Im understanding this correctly you implemented this workload flag so that it has to be set when the k8s connector is installed in the cluster right ? I think we need to change this so that it is part of the kubernetes config coming from iris

Reply:
changed.
Now we are checking if kubernetesConfig.DiscoveryMode == WORKLOAD instead of this flag. Removed the flag const.